scheduler/dirsvc.c

```diff
@@ -440,6 +440,12 @@ dnssdBuildTxtRecord(
     keyvalue[count  ][0] = "pdl";
     keyvalue[count++][1] = p->pdl ? p->pdl : "application/postscript";
 
+    /* iOS 6 does not accept this printer as AirPrint printer if there is
```

Comment:
If we are going to do this, we need to do it right. PAPPL provides the proper mapping from pwg-raster-document-type-supported/resolution-supported/sheet-back to urf-supported. And then the URF key needs to use the value of the urf-supported attribute.